app/components/select_component.html.erb

```diff
@@ -0,0 +1,7 @@
+<div>
```

Comment:
Something to think about is we now have *two* places that we are defining how a `select` field should work.

My preference would be to render the partial here directly; for now; and then inline that partial once we've either:

A) replaced all the `render partial: "select"` with `render SelectComponent`... _or_
B) Did the fancy-pants Custom Form Builder thing I described in the initial comment; and replaced all the `render partial: 'select'` with `f.select`.

Reply:
Yeah that's fair. @zspencer I'm always curious what your restrained, preferred incremental design approach will be and what diverging paths you will suggest for the code I'm madly slinging around. 😅 

Doing `A)` is not a heavy lift at the moment (though let's see how good our test coverage is to catch any mistakes I make!). I designed this select component as a similar base layer component to replace the current partial so that's a reasonable step forward. There aren't too many spots to find/replace.

Reply:
Perfect! We could always go *full* ouroboros and replace the *body* of the render partial with `render SelectComponent` 🤣 😭